docs/user/summary.md

```diff
@@ -52,4 +70,12 @@ which gives:
 
 You can change the boundaries on all of the panels by adding more keys to the dictionary. The keys must be named correctly to work, using the strings at the top of this tutorial.
 
+Other common options include:
```

Comment:
```suggestion
Other common options include:

```
Again need a space or else it breaks in markdown

Reply:
This is still not formatted correctly in the new link, but I'm not sure exactly why?